README.md

```diff
@@ -1423,6 +1423,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Kirtiraj Thakor](https://github.com/kirtiraj22)
   - [Kiruba Muthupalani](https://github.com/kiruba48)
   - [Kishan Singh Thakur](https://github.com/KISHAsingh43)
+  - [Kobin Duyu] (https://github.com/kobbin003)
```

Comment:
```suggestion
  - [Kobin Duyu](https://github.com/kobbin003)
```

Reply:
Sorry I am a bit confused. What should be my next course of action? should I update readme as suggested and pull request?

Reply:
yes :) you did right!